packages/fx-core/src/core/question.ts

```diff
@@ -373,6 +378,10 @@ export function createCapabilityQuestionPreview(inputs?: Inputs): SingleSelectQu
     staticOptions.splice(newBots.length, 0, ExistingTabOptionItem);
   }
 
+  if (isV3Enabled()) {
+    staticOptions.splice(newBots.length, 1);
```

Comment:
Looks tricky. I will prefer something like:
```
const Dashboard = isV3Enabled() ? [] : [ DashboardOptionItem ]
const staticOptions: StaticOptions = [
    ...newBots,
    ...Dashboard,
    ...
  ];
```

Reply:
Nice comment. I've adjusted it.